### DISCUSSION TITLE
code mixed tts

### DISCUSSION THREAD

**Discussion by nellorebhanuteja** [espnet/espnet#4494] [Q&A] [answered]
What are the changes i need to make to training and decoding scripts for creating code mixed TTS ?

**Comment by sw005320** [comment]
This is an interesting but challenging topic.
If we have enough data (and possibly good g2p for both languages), we may just throw it into our default training/inference recipes by using the data.
But we may not have such data.

**Comment by kan-bayashi** [comment, marked as answer]
https://github.com/espnet/espnet/tree/master/egs2/css10/tts1
CSS10 recipe is lang-mixed recipe.
Compared to the mono lang recipe, we need to use phonemized text with g2p=none since our text frontend does not support lang-mixed case.
https://github.com/espnet/espnet/blob/f2778f798b76e102602078213a21b63aa592be70/egs2/css10/tts1/local/data.sh#L102-L121
